react/components/Alert/index.js

```diff
@@ -92,6 +92,8 @@ class Alert extends Component {
             title="Close"
             className={`vtex-alert__close-icon pointer pv2 c-on-base ph3 ${innerVerticalPadding}`}
             onClick={onClose}
+            onKeyPress={onClose}
```

Comment:
shouldn't this be checking for the type of key pressed? like an 'enter' or 'space' 🤔

Reply:
imagine you tabbing into this "button" and pressing tab to focus the next element and the alert closing when you didn't expect it to

Reply:
It really must be done!

Reply:
fixed @lucasecdb